docs/jdbc-doris-catalog.md

```diff
@@ -0,0 +1,173 @@
+---
+title: "Doris catalog"
+slug: /jdbc-doris-catalog
+keywords:
+- jdbc
+- Doris
+- metadata
+license: "Copyright 2024 Datastrato Pvt Ltd.
+This software is licensed under the Apache License version 2."
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Introduction
+
+Gravitino provides the ability to manage Doris metadata.
+
+:::caution
+Gravitino saves some system information in schema and table comment, like `(From Gravitino, DO NOT EDIT: gravitino.v1.uid1078334182909406185)`, please don't change or remove this message.
+:::
+
+## Catalog
+
+### Catalog capabilities
+
+- Gravitino catalog corresponds to the Doris instance.
+- Supports metadata management of Doris (1.2.x).
```

Comment:
Does version `1.2.x` is the mostly used version currently?

Reply:
Doris 2.x version utilizes a vectorized engine. Xiaomi is still in the testing phase and has not officially used it yet. 

Also, there are updates in Doris 2 metadata, such as auto-increment column. 

We could consider supporting Doris 2 later

Reply:
OK, let's change it after we have thoroughly tested other versions.